content/v2.0/query-data/guides/scalar-values.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Extract scalar values in Flux
+description: >
+  placeholder
+menu:
+  v2_0:
+    name: Extract scalar values
+    parent:  How-to guides
+weight: 210
+v2.0/tags: [scalar]
+---
+
+Use Flux [stream and table functions](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/)
+to extract scalar values from Flux query output for things like dynamically setting
+variables using query results.
+
+**To extract scalar values from output:**
+
+1. [Extract a table](#extract-a-table)
+2. Use the extracted table and [extract an array of column values](#extract-a-column-from-the-table)
```

Comment:
From? the extracted table, [extract...

should we make links and headers match here?